odp_sdk/tabular_storage_client.py

```diff
@@ -254,15 +267,128 @@ def delete_stage_request(self, resource_dto: ResourceDto, table_stage: TableStag
         except requests.HTTPError as e:
             if response.status_code == 400:  # This is coming as 400 for both 400 and 404 from ODP
                 raise OdpResourceNotFoundError("Schema not found") from e
+            raise
+
+    def select(self, resource_dto: ResourceDto, filter_query: Optional[dict]) -> List[Dict]:
+        """
+        Select data from dataset
+
+        Args:
+            resource_dto: Dataset manifest
+            filter_query: filter query in OQS format
+
+        Returns:
+            Data that is queried
+
+        Raises
+            OdpResourceNotFoundError: If the schema cannot be found
+        """
+
+        if resource_dto.metadata.uuid:
+            url = f"{self.tabular_storage_url}/{resource_dto.metadata.uuid}/list"
+        else:
+            url = f"{self.tabular_storage_url}/catalog.hubocean.io/dataset/{resource_dto.metadata.name}/list"
+
+        if filter_query:
+            response = self._http_client.post(url, content=filter_query)
```

Comment:
Maybe set `stream=True` in the request?

Reply:
I tis already set as True in API so me not sending it is already same as sending as True. Do you still want me to set it?

Reply:
Setting `stream=True` will control the behavior of the request, not the API :) Streaming is enabled by default in the API, but unless `stream=True`, the request will wait for all data to arrive before returning.

In order to disable streaming in the API, set the header `X-ODP-CHUNKED-ENCODING=false`, however this is not desirable.